vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```
@@ -671,4 +671,9 @@
     </ItemGroup>
   </Target>
   <Target Name="Deploy" />
+  <Target Name="Preprocess">
+    <Message Condition="'$(Source)'!=''" Text="Preprocessing: $(Source)" />
```

> Conditio [](start = 12, length = 9)

```suggestion
 Importance="High" Condition
```
Should we set this to high? I often build with `/v:m`

---

this is just informational so I think it's fine to leave as is, lmk if you disagree